src/map/map_draw.cpp

```diff
@@ -260,21 +260,21 @@ void CViewport::DrawMapGridInViewport() const
 /// Highlight overlays when size of logical and graphical tiles are identical
 void CViewport::HighlightGraphicalTileInViewport(int dx, int dy, uint32_t color /* = ColorRed */, uint8_t transparency /* = 64*/) const
 {
-	Video.FillTransRectangleClip(color, dx, dy, PixelTileSize.x, PixelTileSize.y, transparency);
+    Video.FillTransRectangleClip(color, dx, dy, PixelTileSize.x, PixelTileSize.y, transparency);
```

Comment:
Actually, you break indentation style, we use tab as tabulation, (and space for alignment).
I would probably submit .clangformat now that 3rd party code is in third-party directory :-)